backend/src/main/java/com/google/sps/servlets/SearchMatchServlet.java

```diff
@@ -142,7 +161,12 @@ private void sendMatchResponse(HttpServletResponse response, Match match) throws
     matchExists.put(JSON_FIRST_PARTICIPANT_USERNAME, match.getFirstParticipantUsername());
     matchExists.put(JSON_SECOND_PARTICIPANT_USERNAME, match.getSecondParticipantUsername());
     matchExists.put(JSON_DURATION, match.getDuration());
-
+    try {
+      EMAIL_NOTIFIER.sendMatchEmail(
+          match.getFirstParticipantUsername(), match.getSecondParticipantUsername());
+    } catch (MessagingException | GeneralSecurityException e) {
+      e.printStackTrace();
+    }
```

Comment:
Send email to only the current participant (could be firstParticipant or secondParticipant)! But also yes, pass in the other participant's username (have to figure out if the participant is first or second)

Reply:
Should I add a TODO